frontend/src/components/RegistrationForm.js

```diff
@@ -0,0 +1,107 @@
+import { useState } from "react";
+import { Link } from "react-router-dom";
+
+export default function RegistrationForm() {
+	const [name, setName] = useState("");
+	const [email, setEmail] = useState("");
+	const [password, setPassword] = useState("");
+	const [confirmPassword, setConfirmPassword] = useState("");
+
+	async function signUp(event) {
+		event.preventDefault();
+		let userData = { name, email, password };
+
+		let result = await fetch("http://127.0.0.1:8000/api/register", {
+			method: "POST",
+			body: JSON.stringify(userData),
+			headers: {
+				"Content-Type": "application/json",
+				Accept: "application/json",
+			},
+		});
+
+		result = await result.json();
+		console.log(result);
+	}
```

Comment:
1. the title says markup but you included this integration part.
2. you created an axios base but you used fetch. Utilize the axios